java/google/registry/rde/imports/RdeImportUtils.java

```diff
@@ -66,10 +83,11 @@ public RdeImportUtils(
     this.escrowBucketName = escrowBucketName;
   }
 
-  private <T extends EppResource> void importEppResource(final T resource, final String type) {
+  public <T extends EppResource> ImmutableSet<Object> getIndexesForEppResource(T resource,
+      String type) {
```

Comment:
Why is type being passed in as a String?  What is it exactly, a class name?  It seems to me like you don't even need to pass it, since you can get what you need based on the class that resource is an instance of.